backend/config/corpora-api.yml

```diff
@@ -630,6 +667,30 @@ components:
           type: string
         detail:
           type: string
+    gene_sets:
+      type: object
+      description: User created geneset information
+      properties:
+        gene_set_name:
```

Comment:
```suggestion
        name:
```

Reply:
Timmy is planning on using gene_set_name based on the tidy csv for simplicity